client/packages/system/src/utils.ts

```diff
@@ -153,6 +153,19 @@ export const getInvoiceLocalisationKey = (type: InvoiceNodeType): LocaleKey => {
   }
 };
 
+export const getDonorUpdateKeys = (type: UpdateDonorMethodInput): {label: LocaleKey, message: LocaleKey} => {
```

Comment:
This is just a little helper function because I thought it would be good to have multiple translation strings corresponding to each case of the update method input.